client/my-sites/importer/importing-pane.jsx

```diff
@@ -115,15 +115,8 @@ export const ImportingPane = React.createClass( {
 
 	getHeadingText: function() {
 		return this.translate(
-			'Importing may take a while, but you can ' +
-			'safely navigate away from this page if you need ' +
-			'to. If you {{b}}stop the import{{/b}}, your site ' +
-			'will be {{b2}}partially imported{{/b2}}.', {
-				components: {
-					b: <strong />,
-					b2: <strong />
-				}
-			}
+			'Importing should take 15 minutes, sometimes more if your site has a lot of media. ' +
```

Comment:
How about this 

`Importing takes 15 minutes or a while longer if your site has a lot of media.`

Short, conversation, and there is no comma :)